### DISCUSSION TITLE
1.8之后版本应如何禁用自动将genre添加到tag?

### DISCUSSION THREAD

**Discussion by GRFTSOL** [Yuukiy/JavSP#405] [问答 | Q&A] [answered]
1.7.3可以使用`add_genre_to_tag = no`禁用将genre写入到tag, 1.8.0的配置文件是使用`add_custom_tags = no`来实现吗

**Comment by glyh** [comment, marked as answer]
通过custom_genres_fields和custom_tags_fields
https://github.com/Yuukiy/JavSP/blob/9832462b8fc16cd1112bdf01136ad650bdf9008d/config.yml#L103-L111

**Reply by GRFTSOL** [reply]
好快的回!
可以了
